tensorflow/contrib/quantize/python/quantize.py

```diff
@@ -37,10 +37,10 @@
 _RELU_TYPES = {'Relu', 'Relu6'}
 
 _QUANTIZATION_OP = {'FakeQuantWithMinMaxVars'}
-_VALID_SRC_OP = {'Add', 'AddV2', 'Mul'}
-_INTERMEDIATE_OP = {'Add', 'AddV2', 'Mul'}
+_VALID_SRC_OP = {'Add', 'AddV2', 'Mul', 'Sub', 'ConcatV2'}
```

Comment:
could we add a test for the contrib/quantize changes, and move the contrib/quantize changes to a separate pr from the tflite changes.

Reply:
Sure, I will add some tests for this.

Regarding moving the `contrib/quantize` out, I am not sure about setting min/max of ABS in TOCO, which should be consistent with the TFLite kernel I think.

Reply:
Removed `contrib/quantize` changes.